docs/how-to/set-base-image.md

```diff
@@ -0,0 +1,14 @@
+# How to set base image
+
+This charm supports deploying the runners on different base images.
+
+By using [`juju config`](https://juju.is/docs/juju/juju-config) to change the
+[charm configuration labels](https://charmhub.io/github-runner/configure#base-image), the runner
+can be deployed with a different Ubuntu base image. The supported base images are limited to jammy
+and noble to ensure guaranteed capabilities.
```

Comment:
Perhaps we just note that they are the most recent LTS images or something like that?

Reply:
Updated, thank you!